### PR TITLE
04005 footnote 7304293080 commodity remove

### PR DESCRIPTION
**Pull request by myxrome**
[trello card](https://trello.com/c/IHEQTfqx/950-remove-footnote-that-is-no-longer-needed)
added script to delete footnote '04005' and commodity '7304293080' association.

### CODE REVIEW

**Comment by matthewford** [comment]
@myxrome can we actually end date not delete?

File: db/data_migrations/20200921120101_delete_05004_footnote.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name 'Delete 05044 footnote for 7304293080 commodity'
+
+  up do
+    applicable do
+      FootnoteAssociationGoodsNomenclature::Operation.where(
+        goods_nomenclature_sid: 86621,
+        footnote_id: '004',
+        footnote_type: '05',
+        goods_nomenclature_item_id: '7304293080',
+        operation: 'D'
```
Comment (matthewford) [inline db/data_migrations/20200921120101_delete_05004_footnote.rb:11]: I would create a update operation to just endate the record

**Review by matthewford — CHANGES_REQUESTED** [review]

File: db/data_migrations/20200921120101_delete_05004_footnote.rb (hunk 1 above)
Comment (myxrome) [reply db/data_migrations/20200921120101_delete_05004_footnote.rb]: fixed.

**Comment by myxrome** [comment]
@matthewford I've replaced delete code with association update code.

File: db/data_migrations/20200921120101_delete_05004_footnote.rb (hunk 1 above)
Comment (matthewford) [reply db/data_migrations/20200921120101_delete_05004_footnote.rb]: Just got confirmation the end date was 16.05.2020

**Review by matthewford — APPROVED** [review]

**Pull request merged**